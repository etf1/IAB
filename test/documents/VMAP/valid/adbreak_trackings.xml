<!-- AdBreaks with different Tracking values -->
<vmap:VMAP version='1.0' xmlns:vmap='http://www.iab.net/vmap-1.0'>
    <vmap:AdBreak breakType='linear' timeOffset="start" >
        <vmap:TrackingEvents>
            <vmap:Tracking event='breakStart'>
                <![CDATA[http://7b9df.v.fwmrm.net/ad/1]]>
            </vmap:Tracking>
        </vmap:TrackingEvents>
    </vmap:AdBreak>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="50%" >
        <vmap:TrackingEvents>
            <vmap:Tracking event='breakStart'>
                <![CDATA[http://7b9df.v.fwmrm.net/ad/1]]>
            </vmap:Tracking>
            <vmap:Tracking event='breakEnd'>
                <![CDATA[http://7b9df.v.fwmrm.net/ad/2]]>
            </vmap:Tracking>
            <vmap:Tracking event='error'>http://7b9df.v.fwmrm.net/ad/3</vmap:Tracking>
        </vmap:TrackingEvents>
    </vmap:AdBreak>
    <vmap:AdBreak breakType='linear' timeOffset="end" >
        <vmap:TrackingEvents></vmap:TrackingEvents>
    </vmap:AdBreak>
</vmap:VMAP>
