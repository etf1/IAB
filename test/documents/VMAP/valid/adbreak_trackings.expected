{
  "version": "1.0",
  "breaks": [
    {
      "timeOffset": "start",
      "breakTypes": [
        1
      ],
      "trackings": [
        {
          "level": 1,
          "uri": "http://7b9df.v.fwmrm.net/ad/1"
        }
      ]
    },
    {
      "timeOffset": "50%",
      "breakTypes": [
        1
      ],
      "trackings": [
        {
          "level": 1,
          "uri": "http://7b9df.v.fwmrm.net/ad/1"
        },
        {
          "level": 2,
          "uri": "http://7b9df.v.fwmrm.net/ad/2"
        },
        {
          "level": 3,
          "uri": "http://7b9df.v.fwmrm.net/ad/3"
        }
      ],
      "id": "cue_point-347762684"
    },
    {
      "timeOffset": "end",
      "breakTypes": [
        1
      ]
    }
  ]
}
